lib/autocapture/autocapture_options.dart

```diff
@@ -0,0 +1,64 @@
+import 'attribution_options.dart';
+import 'page_views_options.dart';
+
+/// Options for the autocapture feature.
+///
+/// Currently configuring autocapture is only supported on Web.
+/// Refer to docs for more details.
+///
+/// * [Web](https://amplitude.com/docs/sdks/analytics/browser/browser-sdk-2#autocapture)
+class AutocaptureOptions {
+  /// Web specific
+  ///
+  /// Configures marketing attribution tracking using `AttributionOptions`. See [docs](https://amplitude.com/docs/sdks/analytics/browser/browser-sdk-2#track-marketing-attribution)
+  /// for more information. Set to `false` to disable tracking marketing attribution.
+  ///
+  /// Can be either `AttributionOptions` or `false`.
+  final dynamic attribution;
```

Comment:
How about replacing `dynamic` with sealed class for better type safety? https://dart.dev/language/class-modifiers#sealed

```
sealed class Attribution {}

class AttributionOptions extend Attribution {}

// equivalent of false
class AttributionDisabled extend Attribution {}

// equivalent of true
class AttributionEnabled extend Attribution {}
```